src/subdomains/core/aml/enums/aml-error.enum.ts

```diff
@@ -51,6 +51,7 @@ export enum AmlError {
   CARD_NAME_MISMATCH = 'CardNameMismatch',
   VIDEO_IDENT_MISSING = 'VideoIdentMissing',
   LIQUIDITY_LIMIT_EXCEEDED = 'LiquidityLimitExceeded',
+  IBAN_COUNTRY_NOT_ALLOWED = 'IbanCountryNotAllowed',
 }
```

Comment:
Shouldn't it be `IBAN_CURRENCY_MISMATCH`? The country is not blocked per se, just for the selected currency.